analysis/itdk_geo.py

```diff
@@ -38,8 +42,10 @@ def get_node_ids_with_geo_coordinates():
     node_geo_df = parse_node_geo_as_dataframe()
     return node_geo_df.index.tolist()
 
-def convert_routes_from_ip_to_latlon(routes, node_ip_to_id, node_geo_df, output_file: Optional[str]):
-    logging.info('Converting routes from IPs to lat/lons ...')
+def convert_routes_from_ip_to_latlon(routes, node_ip_to_id, node_geo_df,
```

Comment:
Typing also has List and Tuple.

Reply:
In general I'm supportive of typing all code but I will leave it up to you!

Reply:
Agree. I think this one might be one of the older code that I didn't add type checking. I'll update it.